src/ui/locales/de.js

```diff
@@ -0,0 +1,25 @@
+// @flow
+
+const locale = {
+    "AttributionControl.ToggleAttribution": "Zuordnung umschalten",
```

Comment:
Does this refer to map attribution? If so, I don't think `Zuordung` is an ideal translation here. [Esri](https://support.esri.com/de/technical-article/000012040) uses `Attributierung`

```suggestion
    "AttributionControl.ToggleAttribution": "Attributierung umschalten",
```